catalog/rest/catalog-rest-endpoint/src/test/java/org/codice/ddf/endpoints/rest/RestEndpointIT.java

```diff
@@ -194,6 +194,8 @@ protected BundleOption getBundleOptions() {
         return super.getBundleOptions()
             .add("org.bouncycastle", "bcprov-jdk15on")
             .add("ddf.catalog.transformer", "catalog-transformer-attribute")
+            .add("org.codice.thirdparty", "jts")
```

Comment:
❓Do we know why those would now be needed? It worries me a little that these tests now need to include those bundles (especially `picocontainer`) to be able to run.

Reply:
I pulled this change in from @clockard 's initial changes. I did not try running it without this change in there. He might be able to provide insight.

Reply:
I tried removing those and the tests still passed, so we should take those out since they aren't needed.

Reply:
When I was doing the update, I had changed the third party stuff a lot and I needed to include these here. (I don't remember why though). I think scott's changes are less invasive so these are probably not needed.